tests/e2e/fixtures/statsFixtures.js

```diff
@@ -18,22 +18,21 @@ const seasonFixtures = [
     startTime: dayjs('2022-02-16 18:00').valueOf(),
     endTime: dayjs('2022-03-16 18:00').valueOf(),
   },
+  {
+    name: 'Current Season',
+    startTime: dayjs().subtract(2, 'week').valueOf(),
+    endTime: dayjs().add(12, 'week').valueOf(),
+  },
   {
     name: 'Future Spades Season',
     startTime: dayjs().add(1, 'year').valueOf(),
     endTime: dayjs().add(1, 'year').valueOf(),
   },
   {
-    name: 'World Championship Season',
-    startTime: dayjs().subtract(1, 'second').valueOf(),
-    endTime: dayjs().add(13, 'week').valueOf(),
-    firstPlace: 'player1',
-    secondPlace: 'player2',
-    thirdPlace: 'player3',
-    fourthPlace: 'player4',
-    bracketLink: 'https://github.com/cuttle-cards/cuttle',
-    footageLink: 'https://github.com/cuttle-cards/cuttle-assets',
-  },
+    name: 'World Championship',
+    startTime: dayjs().subtract(10, 'seconds').valueOf(),
+    endTime: dayjs().subtract(9, 'seconds').valueOf(), 
```

Comment:
This is the right idea, but it should not overlap with the current season (atm this takes place in the past, but during a time that the current season was also still active). There should only be one active season at a time
```suggestion
    startTime: dayjs().subtract(2, 'week').subtract(10, 'second').valueOf(),
    endTime: dayjs().subtract(2, 'week').subtract(9, 'second').valueOf(), 
```